README.md

```diff
@@ -1,20 +1,25 @@
 Pretty website for ev3dev [![Build Status](https://travis-ci.org/ev3dev/ev3dev.github.io.svg?branch=master)](https://travis-ci.org/ev3dev/ev3dev.github.io)
 ================
 
-This is the web page at <http://www.ev3dev.org> and also <http://ev3dev.github.io>.
+This is the source repository for the web page at <http://www.ev3dev.org>. If you want
+to learn more about ev3dev, visit the site or check out our [GitHub repo](github.com/ev3dev/ev3dev).
 
-Feel free to fork and make a pull request.
+Feel free to fork and make a pull request! If you're looking for ways to
+contribute, check out issues [tagged "help wanted"](https://github.com/ev3dev/ev3dev.github.io/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22); there are often tasks there
```

Comment:
Clarify here "contribute *to our website*". We also have "help wanted" tags in the main ev3dev issues, so this could be confusing (in fact it did confuse me the first time I looked at it).